### PR TITLE
Allow for setting the priority class and resource requests/limits.

### PR DESCRIPTION
**Pull request by bmcstdio**
This PR adds the ability to set (in the case of the Hubble UI) and override (in the case of Hubble itself) the priority class being used, as well as the ability to set resource requests and limits.

### CODE REVIEW

File: install/kubernetes/hubble/values.yaml [hunk 1]
```diff
@@ -49,6 +49,8 @@ ui:
     pullPolicy: Always
   clusterDomain: cluster.local
   replicas: 1
+  resources: {}
```
Comment (michi-covalent) [inline install/kubernetes/hubble/values.yaml]: could you add a comment here with a sample value?

File: install/kubernetes/hubble/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: hubble
-version: 0.0.1
+version: 0.0.2
```
Comment (glibsm) [inline install/kubernetes/hubble/Chart.yaml]: We haven't been very diligent about actually nudging this in the past, thanks for pointing this out.

File: install/kubernetes/hubble/values.yaml [hunk 3]
```diff
@@ -62,3 +64,6 @@ ingress:
   #  - secretName: chart-example-tls
   #    hosts:
   #      - chart-example.local
+
+resources: {}
+priorityClassName: system-node-critical
```
Comment (Rolinh) [inline install/kubernetes/hubble/values.yaml]: Would you mind adding a comment here? Something like the following:
> The priority class `system-node-critical`  marks  add-on pods as critical to the node itself. This priority class is only valid under the `kube-system` namespace.

**Comment by bmcstdio** [comment]
@michi-covalent @glibsm @Rolinh thank you for your reviews and suggestions. I've addressed all your comments.

**Pull request merged**